README.rst

```diff
@@ -11,18 +11,43 @@ Blosc filter for HDF5
 .. |powered| image:: http://b.repl.ca/v1/Powered--By-Blosc-blue.png
         :target: https://blosc.org
 
-This is an example of filter for HDF5 that uses the Blosc compressor.
+This is a filter for HDF5 that uses the Blosc compressor; by installing this
+filter, you can read and write HDF5 files with Blosc-compressed datasets.
 
 You need to be a bit careful before using this filter because you
 should not activate the shuffle right in HDF5, but rather from Blosc
 itself.  This is because Blosc uses an SIMD shuffle internally which
 is much faster.
 
+Installing the Blosc filter plugin
+==================================
 
-Using the Blosc filter from HDF5
-================================
+Instead of just linking this Blosc filter into your HDF5 application, it is possible to install
+it as a system-wide HDF5 plugin (with HDF5 1.8.11 or later).  This is useful because it allows
+*every* HDF5-using program on your system to transparently read Blosc-compressed HDF5 files.
 
-In order to register Blosc into your HDF5 application, you only need
+As described in the `HDF5 plugin documentation <https://www.hdfgroup.org/HDF5/doc/Advanced/DynamicallyLoadedFilters/HDF5DynamicallyLoadedFilters.pdf>`_, you just need to compile the Blosc plugin into a shared library and
+copy it to the plugin directory (which defaults to ``/usr/local/hdf5/lib/plugin`` on non-Windows systems).
+
+Following the ``cmake`` instructions below produces a ``blosc_plugin.so`` shared library 
```

Comment:
It doesn't look like `cmake` build generates the plugin shared library, only the filter.